### DISCUSSION TITLE
Login stuck

### DISCUSSION THREAD

**Discussion by bentinata** [dmadisetti/steam-tui#56] [Q&A] [answered]
Hi, just trying it out and it stuck right on login. Tried putting my username then `Enter`, and putting my username directly on `default_user` on `.config/steam-tui/config.json`.

<img width="1084" alt="Screenshot 2023-04-22 at 09 27 15" src="https://user-images.githubusercontent.com/3183661/233757779-032d601e-5ea0-42de-b43b-fe27dea52117.png">

Some other info:
- `steamcmd` `login` works (it shows my friend list when I do `user_friends`)
- Current OS is macOS Ventura 13.3.1, Intel processor
- I installed from `cargo` (`cargo 1.68.1 (115f34552 2023-02-26)`)
- `Ctrl-C` does nothing, need to send `SIGTERM` for `steam-tui` to exit

**Comment by dmadisetti** [comment]
It might just be updating? What happens if you leave it for 10-20 minutes?

**Reply by bentinata** [reply]
I waited for an hour and still updating. Is there a way to debug?

**Reply by dmadisetti** [reply]
Yes, `steam-tui 2> error.log`

**Reply by bentinata** [reply]
It doesn't output anything to stderr.

**Reply by bentinata** [reply]
By the way, this is what happen when I kill the `steam-tui` process. So yeah, it's calling `steamcmd`.

```sh
$ ps ax | rg steam
 9112   ??  S      0:00.01 bash /usr/local/Caskroom/steamcmd/1679680174/MacOS/steamcmd.sh +@ShutdownOnFailedCommand 0 +@NoPromptForPassword 1
 9116   ??  S      0:03.91 /usr/local/Caskroom/steamcmd/1679680174/MacOS/steamcmd +@ShutdownOnFailedCommand 0 +@NoPromptForPassword 1
 9280 s003  R+     0:00.00 rg steam
 ```

**Reply by dmadisetti** [reply]
What happens running `usr/local/Caskroom/steamcmd/1679680174/MacOS/steamcmd +@ShutdownOnFailedCommand 0 +@NoPromptForPassword 1` directly? Could be that the process is stuck

**Reply by bentinata** [reply]
Tried, doesn't seem to be different from running `steamcmd`.

**Reply by dmadisetti** [reply]
So running the command directly does not stick?

What version of steam-tui are you using?

**Comment by dmadisetti** [comment, marked as answer]
Closing for https://github.com/dmadisetti/steam-tui/issues/54

Let's put further conversation there
